static/app/views/performance/transactionSummary/header.tsx

```diff
@@ -68,6 +70,15 @@ class TransactionHeader extends React.Component<Props> {
     // TODO(k-fish): Add analytics for tags
   };
 
+  trackEventsTabClick = () => {
+    const {organization} = this.props;
+    trackAnalyticsEvent({
+      eventKey: 'performance_views.events.events_tab_clicked',
```

Comment:
Are these new analytics events? You might need to add them to the `reload` repo too ([example](https://github.com/getsentry/reload/pull/206/files))

Reply:
Ahh good to know, thanks Shruthi!
Opened up the pr in `reload` here:
https://github.com/getsentry/reload/pull/208